statsmodels/tsa/tests/test_arima_process.py

```diff
@@ -73,7 +73,25 @@ def test_arma_acf():
     # rep 2: manually
     acovf = np.array([1. * sigma * phi ** i / (1 - phi ** 2) for i in range(N)])
     rep2 = acovf / (1. / (1 - phi ** 2))
-    assert_almost_equal(rep1, rep2, 8)  # 8 is max precision here
+    assert_allclose(rep1, rep2)
+
+    # Test other cases, against R's ARMAacf
```

Comment:
```suggestion
def test_arma_acf_compare_R_ARMAacf():
    # Test other cases, against R's ARMAacf; GH#5324
```

Reply:
Why the reference to the github PR?

Reply:
Habit.  Avoids the need to look up git history.